### ISSUE TITLE
Essential stats volleyball

### ISSUE THREAD

**Issue by mchiapello** [openvolley/datavolley#6]
I tryed to scout a fake game with Essential stats volleyball (Data project family). The software is free and super easy to use, perfect for my team.

I had two problems:

#### When I read the dvw file I got the following warnings:

```
x <- dv_read("~/Downloads/&PGS-Avv&Scout.dvw", insert_technical_timeouts = FALSE)
Warning messages:
1: Unknown or uninitialised column: `number`. 
2: Unknown or uninitialised column: `number`. 
3: Unknown or uninitialised column: `number`. 
4: Unknown or uninitialised column: `number`. 
5: Unknown or uninitialised column: `number`. 
6: Unknown or uninitialised column: `number`. 
7: Unknown or uninitialised column: `number`. 
8: Unknown or uninitialised column: `special_role`. 
```

It is something I have to worry about?

#### When I tried to generate a report I got an error

```
vr_match_summary(x)

Quitting from lines 135-141 (html.Rmd) 
Error in `dplyr::select()`:
! Column `number` not found in `.data`.
Backtrace:
  1. volleyreport:::vr_content_team_table(vsx, kable_format, which_team = "visiting")
 36. rlang:::abort_data_pronoun(x, call = y)
```

Here you can find the file I read in. I just changed the extention from dvw to txt to be able to load it onto Github [&PGS-Avv&Scout.txt](https://github.com/openvolley/datavolley/files/8399212/PGS-Avv.Scout.txt)

**Comment by raymondben** [comment]
Hi @mchiapello, thanks for the note! The warnings are fixed in v1.2.6 ("fixed" in the sense that the code no longer tries to manipulate non-existent columns in the data). However, the warnings were reflecting a deeper problem, which is that while Essential Stats is very convenient, it's unfortunately quite limited in what it records. Notably:

- you can only enter the player list for one team (check the `x$meta$players_v` table - it's empty because there is no player information for the visiting team in the dvw file)
- you can't enter the actual on-court lineup for either team (columns `plays(x)$home_p1` and similar are all NA, we don't know who was in what position on court)
- no skill types (e.g. high ball attack vs quick attack) or locations can be entered
- no non-terminal actions are scouted. Only points and errors

Once we move beyond very basic stats, this level of data capture isn't adequate. A big issue is the lack of non-terminal actions (so we can't calculate e.g. attack kill % or efficiency, because we don't know how many attempts were made). Similarly we can't assess reception performance because we don't have grades for each reception made (perfect/good/OK/poor) - we only know about reception errors. And many more things. So the failure of `vr_match_summary` is unavoidable - most of our downstream analysis won't work with Essential Stats files. (We should perhaps issue a warning with `dv_read` when it reads such files?)

So, summary: `dv_read` will now read these, but ES is probably not the scouting software to use. I'll leave the issue open to perhaps add a warning as mentioned above.

**Comment by raymondben** [comment]
Also noting, ES has been officially retired now: https://volleyballsupport.geniussports.com/en/support/solutions/articles/9000214931-end-of-life-announcement-essential-stats
